tools/file_packager.py

```diff
@@ -929,7 +929,7 @@ def generate_js(data_target, data_files, metadata):
     node_support_code = ''
     if options.support_node:
       node_support_code = '''
-        if (typeof process === 'object' && typeof process.versions === 'object' && typeof process.versions.node === 'string') {
+        if (typeof process === 'object' && typeof !process.browser && typeof process.versions === 'object' && typeof process.versions.node === 'string') {
```

Comment:
Looks like you have an extra `typeof` here.

Reply:
I just found out， I have remove it.